Domain Services/Health/Expense-v1.1.0.yml

```
@@ -0,0 +1,654 @@
+        referenceId:
+          description: Pass reference to link payees to business entity
+          type: string
+          minLength: 2
+          maxLength: 64
+          example: WAGESEEKER-01
+        paymentStatus:
+          $ref: '#/components/schemas/PaymentStatus'
+        status:
+          $ref: '#/components/schemas/Status'
+        totalAmount:
+          description: total Amount for the particular bill-detail during payment  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)